libcontainer/nsenter/nsexec.c

```diff
@@ -1053,6 +1165,22 @@ void nsexec(void)
 					bail("failed to sync with parent: SYNC_MOUNTSOURCES_ACK: got %u", s);
 			}
 
+			if (config.idmapsources) {
+				write_log(DEBUG, "request stage-0 to send idmap sources");
+				s = SYNC_MOUNT_IDMAP_PLS;
+				if (write(syncfd, &s, sizeof(s)) != sizeof(s))
+					bail("failed to sync with parent: write(SYNC_MOUNT_IDMAP_PLS)");
+
+				/* Receive and install all idmap fds. */
+				receive_idmapsources(syncfd);
+				write_log(DEBUG, "received opened idmap sources");
```

Comment:
@rata A small suggestion: I think you used this log when you were debugging. If you want to remain this debug log, please add `stage-xxx` in the head. But I suggest to remove this log to keep the same log as `mount sources`.

Reply:
Deleted this one too, thanks!